.github/workflows/build.yml

```diff
@@ -277,6 +297,20 @@ jobs:
           zip_artifact_name: ${{ env.PKG_NAME }}_${{ needs.product-metadata.outputs.product-version }}_linux_${{ matrix.arch }}.zip
           redhat_tag: scan.connect.redhat.com/ospid-f0a92725-d8c6-4023-9a87-ba785b94c3fd/${{env.repo}}:${{env.version}}-ubi
 
+# temporarily disabled till we find out more about the UBI images
```

Comment:
Probably shouldn't put this here, unfortunately. I would argue for taking this snipped and putting it into a card for the backlog for if/when we get around to this.

Reply:
good call, I added: QT-347.